infrastructure/ansible/roles/nginx/templates/etc_nginx_sites_enabled_default.j2

```diff
@@ -26,9 +24,16 @@ server {
       proxy_set_header        X-Forwarded-Proto $scheme;
 
       # Fix the “It appears that your reverse proxy set up is broken" error.
-      proxy_pass          http://localhost:4567;
+      proxy_pass          http://localhost:8080;
       proxy_read_timeout  90;
 
-      proxy_redirect      http://localhost:443 https://localhost:{{ http_server_port }};
+      proxy_redirect      https://localhost:443 https://localhost:{{ http_server_port }};
```

Comment:
One thing to consider: This reverse proxy continues to be a proxy, so I believe lobby ip records etc. will point to the nginx proxy instead of the "actual source", so reintroducing the X-Forwarded-For logic might be necessary here, to let nginx tell the server which was the original IP.

Reply:
Cool, let's keep as advisory and see what happens when it's deployed, we'll know pretty quickly if we need the x-forwarded port if the access-log IPs are incorrect.